docs/schedules.md

```diff
@@ -141,3 +142,24 @@ database for Schedules where `Schedule.next_run` is between
 `scheduler_last_runtime()` and `utcnow()`.  For each of these, a new job is
 launched, and `Schedule.next_run` is changed to the next chronological datetime
 in the list of all occurences.
+
+
+## Complex RRULE Examples
+
+Every day except for April 30th:
+
+    DTSTART;TZID=US/Eastern:20230428T170000 RRULE:INTERVAL=1;FREQ=DAILY EXRULE:INTERVAL=1;FREQ=DAILY;BYMONTH=4;BYMONTHDAY=30
+
+Every 5 minutes but not on Mondays from 5-7pm:
+
+    DTSTART;TZID=America/New_York:20220418T164500 RRULE:INTERVAL=5;FREQ=MINUTELY EXRULE:FREQ=MINUTELY;INTERVAL=5;BYDAY=MO;BYHOUR=17,18
+
+Every 15 minutes Monday to Friday from 10:01am to 6:02pm (inclusive):
+
+    DTSTART;TZID=America/New_York:20220417T100100 RRULE:INTERVAL=15;FREQ=MINUTELY;BYDAY=MO,TU,WE,TH,FR;BYHOUR=10,11,12,13,14,15,16,17,18 EXRULE:INTERVAL=15;FREQ=MINUTELY;BYDAY=MO,TU,WE,TH,FR;BYHOUR=18;BYMINUTE=3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,34,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59
+
+Any Saturday whose month day is between 12 and 18:
+
+    DTSTART:20191219T130551Z RRULE:FREQ=MONTHLY;INTERVAL=1;BYDAY=SA;BYMONTHDAY=12,13,14,15,16,17,18
+
+
```

Comment:
:+1: great example content, thanks for adding those.

Reply:
I'll be adding these to the unit tests as well.